link/uprobe.go

```diff
@@ -149,6 +149,13 @@ func (ex *Executable) uprobe(symbol string, prog *ebpf.Program, ret bool) (*perf
 		return nil, fmt.Errorf("symbol '%s' not found in '%s': %w", symbol, ex.path, err)
 	}
 
+	// Symbols with location 0 from section undef are shared library calls and
```

Comment:
I think this is worth documenting on the Uprobe function as well.

Reply:
Good point, done :+1: